src/store/reducers/common.js

```diff
@@ -238,41 +239,72 @@ export default (state = initialState, action) => {
         logInStatus: action.payload,
       };
     case 'SHOPLIST_SORT_PRODUCT': {
-      const { id } = action;
+      const { id, products: productData } = action;
       const { filterPrice } = state;
       const productState = state?.popularProducts;
-      const productSort = productState?.filter(
-        (prod) =>
-          filterPrice[0] <= prod.price &&
-          prod.price <= filterPrice[1] &&
-          (prod.category === id || prod.color === id)
-      );
+      let productSort;
+      if (productData) {
+        productSort = productData?.filter(
+          (prod) => filterPrice[0] <= prod.price && prod.price <= filterPrice[1]
```

Comment:
What is `filterPrice[0]` and `filterPrice[1]`? Should it be `const [fromPrice, toPrice] = filterPrice;`?